mantidproject/mantid
# Load most recent calibration on EnggDiffUI

**Description of work.**
Save the calibration most recently created or loaded, and load the path to this into the load box on loading the interface.
Also fixes the default path for the shipped full instrument calibration in the settings, and cover this with a unit test

**To test:**
1. Create or load a calibration
2. Close the interface
3. Open it again, the path to the calibration should be in the load box ready to go

Fixes #31907. 






---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
